src/plugins/intel_gpu/tests/unit/test_cases/scatter_nd_update_gpu_test.cpp

```diff
@@ -4412,7 +4412,7 @@ TEST(scatter_nd_update_gpu, dynamic) {
     });
 
     set_values(input2, {
-        0.f, 1.f, 1.f, 2.f, 2.f, 2.f
+        0.f, 0.f, 1.f, 1.f, 0.f, 1.f
```

Comment:
why test is changed?

Reply:
input2 is the indices includes coordinates. data shape is 2x1x2x8. '0,1,1' and '2,2,2' are invalid values.